function-logs/handler.go

```diff
@@ -28,6 +32,16 @@ func Handle(req []byte) string {
 			if len(userQuery) > 0 {
 				user = userQuery
 			}
+			numLines = vals.Get("numLines")
```

Comment:
this should be a `:=` too, then `vat numLines string` and `var since int` are not required.

Reply:
We need these vars out of scope of that if condition, therefore its like that